markdown-mode.el

```diff
@@ -6527,6 +6527,46 @@ BEG and END are the limits of scanned region."
        (progn (goto-char end) (forward-line 1) (point))))))
 
 
+;;; Display inline image =================================================
+
+(defvar markdown-inline-image-overlays nil)
+(make-variable-buffer-local 'markdown-inline-image-overlays)
+
+(defun markdown-remove-inline-images ()
+  (interactive)
+  (mapc 'delete-overlay markdown-inline-image-overlays)
+  (setq markdown-inline-image-overlays nil))
+
+(defun markdown-display-inline-images ()
+  (interactive)
+  (unless (display-graphic-p)
+    (error "Cannot show images."))
+  (save-excursion
+    (save-restriction
+      (widen)
```

Comment:
I took most of code from org-mode and org inline image commands uses `widen`. Commands which I add works same as org-mode commands(`org-toggle-inline-images` etc).